src/components/EmptyThoughtspace.tsx

```diff
@@ -55,9 +55,12 @@ const EmptyThoughtspace = ({ isTutorial }: { isTutorial?: boolean }) => {
           <>
             <span className={css({ userSelect: 'none' })}>
               {isTouch ? (
-                <span>
+                <span style={{ display: 'inline-block', marginTop: '10px' }}>
                   Swipe{' '}
                   <GestureDiagram
+                    cssRaw={css.raw({
+                      marginBottom: '-10px',
+                    })}
```

Comment:
While this does eliminate `!important`, I feel like we're getting further away from the correct solution. We shouldn't have to override multiple layers of styles in order to shift the diagram down. The original `position: relative` and `top: 10px` achieved this with fewer side effects.